docs/workflow/building/libraries/README.md

```diff
@@ -256,3 +256,7 @@ Same as for `dotnet build` or `dotnet publish`, you can specify the desired conf
 ```cmd
 dotnet.cmd pack src\libraries\System.Text.Json\src\ -c Release
 ```
+
+If changes to the library include any API incompatibilities, calling `dotnet pack` will result in API compatibility errors.
```

Comment:
@ViktorHofer will this work for framework libraries like System.Runtime / System.Net.Http / etc?  Do we need to cover that case separately?

Reply:
Non packable projects under src/libraries run APICompat as part of the `dotnet build` command.